metadata-ingestion/src/datahub/api/entities/datajob/dataflow.py

```diff
@@ -120,6 +120,7 @@ def generate_mce(self) -> MetadataChangeEventClass:
                         description=self.description,
                         customProperties=self.properties,
                         externalUrl=self.url,
+                        env=self.cluster,
```

Comment:
is it guaranteed that cluster will map cleanly into the env enum?

Reply:
We always tell everyone to use Fabric type for clusters.

Reply:
Ran this code 
```
import datahub.emitter.mce_builder as builder
from datahub.emitter.mcp import MetadataChangeProposalWrapper
from datahub.emitter.rest_emitter import DatahubRestEmitter
from datahub.api.entities.datajob import DataFlow, DataJob

emitter = DatahubRestEmitter("http://localhost:8080")

jobFlow = DataFlow(cluster="PROD", orchestrator="airflow", id="aseemtest")
jobFlow.emit(emitter)
emitter.emit(jobFlow.generate_mce())


jobFlow = DataFlow(cluster="randomcluster", orchestrator="airflow", id="aseemtest2")
jobFlow.emit(emitter)
emitter.emit(jobFlow.generate_mce())
```
and got this
![image](https://github.com/datahub-project/datahub/assets/4127841/9f83c219-6675-47b6-9a8c-126a7ceb6a71)

As expected cluster with some random cluster will fail. So if folks want to use the environment filter then it needs to  be valid env type. I'll make a change so it logs a warning instead of throwing exception.

Reply:
Pushed commit

[`3b77c66` (#10814)](https://github.com/datahub-project/datahub/pull/10814/commits/3b77c66f2ea6d5a782add06980b97ee8db92842c)